lighthouse-core/test/audits/autocomplete-test.js

```diff
@@ -112,7 +112,7 @@ describe('Best Practices: autocomplete audit', () => {
           snippet: '<input type="text" name="name_cc" autocomplete="namez">',
           type: 'node',
         },
-        suggestion: 'Requires manual review.',
+        suggestion: 'lighthouse-core/audits/autocomplete.js | manualReview # 0',
```

Comment:
here and everywhere an i18n string is being asserted, can you replace with `toBeDisplayString` ?

ex https://github.com/GoogleChrome/lighthouse/blob/5acd30fde3055c5e623484b3648b814b028c9f40/lighthouse-core/test/audits/is-on-https-test.js#L78
https://github.com/GoogleChrome/lighthouse/blob/c78e3e6f3cd5570b24c5c9586832477758c723f3/lighthouse-core/test/runner-test.js#L717

Reply:
on it